apps/dashboard/src/components/welcome/framework-guides.instructions.tsx

```diff
@@ -45,17 +45,28 @@ export const commonInstallStep = (packageName: string): InstallationStep => ({
   codeTitle: 'Terminal',
 });
 
-export const frameworks: Framework[] = [
+export const commonCLIInstallStep = (packageName: string): InstallationStep => ({
+  title: 'Run the CLI command in an existing project',
+  description: `You'll notice a new folder in your project called inbox. This is where you'll find the inbox component boilerplate code. \n You can customize the <Inbox /> component to match your app theme.`,
+  code: `npx novu@add-inbox --appId YOUR_APPLICATION_IDENTIFIER --subscriberId YOUR_SUBSCRIBER_ID`,
+  codeLanguage: 'shell',
+  codeTitle: 'Terminal',
+});
```

Comment:
If we are in EU, we need to add add --region=eu and the cli needs to add the proper flags to properly work during the EU onboarding @iampearceman